packages/coil-extension/src/popup/mocks/loadMockedStates.tsx

```diff
@@ -116,7 +127,8 @@ const MOCK_STATES = [
   { name: 'Start Exploring', state: startExploring },
   { name: 'Paying', state: payingNonCoilSite },
   { name: 'Welcome To Coil', state: welcomeToCoil },
-  { name: 'Alice Unsubscribed', state: aliceUnsubscribed }
+  { name: 'Alice Unsubscribed', state: aliceUnsubscribed },
+  { name: 'Paying Youtube', state: payingYouTube }
```

Comment:
Does this handle Twitch as well since its an adapted site?

Reply:
Yes